internal/models/makemetric.go

```diff
@@ -78,84 +76,6 @@ func makemetric(
 		}
 	}
 
-	for k, v := range tags {
-		if strings.HasSuffix(k, `\`) {
-			log.Printf("D! Measurement [%s] tag [%s] "+
-				"ends with a backslash, skipping", measurement, k)
-			delete(tags, k)
-			continue
-		} else if strings.HasSuffix(v, `\`) {
-			log.Printf("D! Measurement [%s] tag [%s] has a value "+
-				"ending with a backslash, skipping", measurement, k)
-			delete(tags, k)
-			continue
-		}
-	}
-
-	for k, v := range fields {
-		if strings.HasSuffix(k, `\`) {
-			log.Printf("D! Measurement [%s] field [%s] "+
-				"ends with a backslash, skipping", measurement, k)
-			delete(fields, k)
-			continue
-		}
-		// Validate uint64 and float64 fields
-		// convert all int & uint types to int64
-		switch val := v.(type) {
-		case nil:
-			// delete nil fields
-			delete(fields, k)
-		case uint:
-			fields[k] = int64(val)
-			continue
-		case uint8:
-			fields[k] = int64(val)
-			continue
-		case uint16:
-			fields[k] = int64(val)
-			continue
-		case uint32:
-			fields[k] = int64(val)
-			continue
-		case int:
-			fields[k] = int64(val)
-			continue
-		case int8:
-			fields[k] = int64(val)
-			continue
-		case int16:
-			fields[k] = int64(val)
-			continue
-		case int32:
-			fields[k] = int64(val)
-			continue
-		case uint64:
-			// InfluxDB does not support writing uint64
-			if val < uint64(9223372036854775808) {
-				fields[k] = int64(val)
-			} else {
-				fields[k] = int64(9223372036854775807)
-			}
-			continue
-		case float32:
-			fields[k] = float64(val)
-			continue
-		case float64:
```

Comment:
Sorry for opening an old issue but this check was blocking a lot of garbage from being attempted to be written to our output. Is there a different place that this same kind of filtering can be done?

Reply:
Can you describe what you are seeing now in a new issue?